manifests/init.pp

```diff
@@ -46,47 +47,48 @@
 # @param renew_cron_ensure Intended state of the cron resource running certbot renew.
 # @param renew_cron_hour
 #   Optional string, integer or array of hour(s) the renewal command should run.
-#   E.g. '[0,12]' to execute at midnight and midday. Default: fqdn-seeded random
+#   E.g. '[0,12]' to execute at midnight and midday.
 #   hour.
 # @param renew_cron_minute
 #   Optional string, integer or array of minute(s) the renewal command should
-#   run. E.g. 0 or '00' or [0,30]. Default: fqdn-seeded random minute.
+#   run. E.g. 0 or '00' or [0,30].
 # @param renew_cron_monthday
 #   Optional string, integer or array of monthday(s) the renewal command should
-#   run. E.g. '2-30/2' to run on even days. Default: Every day.
+#   run. E.g. '2-30/2' to run on even days.
 #
 class letsencrypt (
   Optional[String] $email                = undef,
-  String $path                           = $letsencrypt::params::path,
-  $venv_path                             = $letsencrypt::params::venv_path,
+  String $path                           = '/opt/letsencrypt',
+  $venv_path                             = '/opt/letsencrypt/.venv',
   Array $environment                     = [],
-  String $repo                           = $letsencrypt::params::repo,
-  String $version                        = $letsencrypt::params::version,
-  String $package_name                   = $letsencrypt::params::package_name,
-  $package_ensure                        = $letsencrypt::params::package_ensure,
-  String $package_command                = $letsencrypt::params::package_command,
-  String $config_file                    = $letsencrypt::params::config_file,
-  Hash $config                           = $letsencrypt::params::config,
-  String $cron_scripts_path              = $letsencrypt::params::cron_scripts_path,
-  Boolean $manage_config                 = $letsencrypt::params::manage_config,
-  Boolean $manage_install                = $letsencrypt::params::manage_install,
-  Boolean $manage_dependencies           = $letsencrypt::params::manage_dependencies,
-  Boolean $configure_epel                = $letsencrypt::params::configure_epel,
-  Enum['package', 'vcs'] $install_method = $letsencrypt::params::install_method,
-  Boolean $agree_tos                     = $letsencrypt::params::agree_tos,
-  Boolean $unsafe_registration           = $letsencrypt::params::unsafe_registration,
-  Stdlib::Unixpath $config_dir           = $letsencrypt::params::config_dir,
+  String $repo                           = 'https://github.com/certbot/certbot.git',
+  String $version                        = 'v0.30.2',
+  String $package_name                   = 'letsencrypt',
+  $package_ensure                        = 'installed',
+  String $package_command                = 'letsencrypt',
```

Comment:
Only VCS has letsencrypt but the repo is named certbot so I think that's actually a safe default. Current master even lacks an entry point for it.